domains/yajtpg.json

```diff
@@ -0,0 +1,14 @@
+{
+   "description":"YajTPG's website",
+   "repo":"https://github.com/YajTPG",
+   "owner":{
+      "username":"YajTPG",
+      "email":"yajtpg@gmail.com"
+   },
+   "record":{
+      "CNAME":"ns101.cloudns.net",
+      "CNAME":"ns102.cloudns.net",
+      "CNAME":"ns103.cloudns.net",
+      "CNAME":"ns104.cloudns.net"
```

Comment:
You can't have multiple cnames. The values look like nameservers but nameserver records are not supported. The supported record types are `CNAME` (single hostname), `A` (a single or a group of ips) and `URL` (simple redirection to any url).

Reply:
Oh, sorry about that. I've updated the file.